apps/web/src/app/settings/verify-email.component.html

```diff
@@ -1,21 +1,15 @@
-<div class="card border-warning">
-  <div class="card-header bg-warning text-white">
+<div
+  class="tw-relative tw-flex tw-min-w-0 tw-flex-col tw-break-words tw-rounded tw-border tw-border-solid tw-border-warning-500 tw-bg-background tw-bg-clip-border"
+>
```

Comment:
I think we can simplify this into

```suggestion
<div class="tw-rounded tw-border tw-border-solid tw-border-warning-500 tw-bg-background">
```

Reply:
Updated.